src/Utilities/ErrorHandling/FormatStacktrace.cpp

```diff
@@ -0,0 +1,39 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Utilities/ErrorHandling/FormatStacktrace.hpp"
+
+#include <boost/algorithm/string.hpp>  // For boost::to_lower
+#include <cstddef>
+#include <cstdlib>  // For std::getenv
+#include <string>
+
+namespace detail_stacktrace {
+
+bool env_var_to_bool(std::string env_var) {
+  boost::to_lower(env_var);
+  return not(env_var == "false" or env_var == "no");
+}
+
+std::string abbreviated_symbol_name(const std::string& symbol_name) {
+  constexpr size_t abbrev_length_begin = 30;
+  constexpr size_t abbrev_length_end = 1;
```

Comment:
Just noting that these changed by one and two orders of magnitude from the previous version.  Was that intentional?

Reply:
Oh sorry, this was just for testing